keps/sig-storage/3333-reconcile-default-storage-class/kep.yaml

```diff
@@ -16,12 +16,12 @@ see-also:
 replaces:
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
```

Comment:
You also need to extend this file:
https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-storage/3333.yaml

Also please ping me for PRR approval once you have SIG-level approval.

Reply:
Ack, added `prr-approvers`.

Reply:
Please remove it from here - you only need: keps/prod-readiness/sig-storage/3333.yaml

[In fact, we're getting rid of this field in kep.yaml to avoid duplication]